ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -9,15 +9,30 @@ proxy:
 metadata:
   name: {{ cluster }}
 networking:
-  machineCIDR: {{ extcidrnet }}
+{% if not dualstack_baremetal|bool %}
+  machineCIDR: {{ extcidrnet[0] }}
+{% endif %}
   networkType: {{ network_type }}
-{% if ipv6_enabled|bool and not ipv4_baremetal|bool %}
+{% if ipv6_enabled|bool and not (ipv4_baremetal|bool or dualstack_baremetal|bool) %}
   clusterNetwork:
   - cidr: fd01::/48
     hostPrefix: 64
   serviceNetwork:
   - fd02::/112
 {% endif %}
+{% if ipv6_enabled|bool and dualstack_baremetal|bool %}
+  machineNetwork:
+  - cidr: {{ extcidrnet[0] }}
+  - cidr: {{ extcidrnet[1] }}
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd02::/48
+    hostPrefix: 64
+  serviceNetwork:
+  - 172.30.0.0/16
+  - fd03::/112
```

Comment:
Same hardcoding here and using different ranges to the one we had in line 21

Reply:
It doesn't really matter what range we use here as far as I understand with regards to being different from line 21 (as long as they are valid ranges and the installer will barf otherwise).  clusterNetwork for ipv6 is hardcoded even now right?

Reply:
yea @smalleni cluster n/w for ipv6 is hardcoded even now 

```
 - cidr: fd02::/48
    hostPrefix: 64
```